pkg/kubelet/pluginmanager/reconciler/reconciler_test.go

```diff
@@ -187,11 +188,14 @@ func Test_Run_Positive_Register(t *testing.T) {
 	stopChan := make(chan struct{})
 	defer close(stopChan)
 	go reconciler.Run(stopChan)
-	socketPath := fmt.Sprintf("%s/plugin.sock", socketDir)
+	socketPath := filepath.Join(socketDir, "plugin.sock")
 	pluginName := fmt.Sprintf("example-plugin")
 	p := pluginwatcher.NewTestExamplePlugin(pluginName, registerapi.DevicePlugin, socketPath, supportedVersions...)
 	require.NoError(t, p.Serve("v1beta1", "v1beta2"))
-	timestampBeforeRegistration := time.Now()
+	// On Windows, time.Now() is not as precise, which means that 2 consecutive calls may
```

Comment:
Indeed, the clock resolution is not as fine grained on Windows as on Linux. Because of this, if a plugin has been reregsitered within a 1-15ms window, the reregistration workflow won't be applied, it won't be detected as being reregistered.

I've moved this to a separate PR, where I'll looking more into it: https://github.com/kubernetes/kubernetes/pull/113253